resources/views/users/print.blade.php

```diff
@@ -352,25 +359,44 @@ class="snipe-table table table-striped inventory"
         </table>
     @endif
 
+    <p></p>
+    <div class="pull-right">
+        <button class="btn btn-default" type="button" data-toggle="collapse" data-target="#eula-row" aria-expanded="false" aria-controls="eula-row" title="EULAs">
```

Comment:
Will this button show up on the print version?

Reply:
@snipe I was in doubt if hide or not but I have seen that the other 2 buttons in the page are printable, so I didn't use a different behavior, but I prefer to hide them. So if you agree I hide all 3 buttons with the property hidden-print